builder/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+docker run -v /var/run/docker.sock:/var/run/docker.sock -v ${PATH_TO_CODE}:/build -e APPLICATION_SLUG=${APPLICATION_SLUG} -it soapbox
```

Comment:
Does it make sense to add the `--rm` flag here to scrap the build container after the process finishes?